taskcluster/fxci_upgrades.rst

```diff
@@ -0,0 +1,57 @@
+FirefoxCI cluster upgrades
+==========================
+
+As of H1 2022, the Taskcluster, Cloudops, and Releng teams have committed to rolling out upgrades to the FirefoxCI cluster at a regular cadence.
+
+Process
+-------
+
+We now have a regular cadence of scheduled upgrades, as noted in the `Public - FirefoxCI Cluster Taskcluster Upgrades calendar`_.
```

Comment:
nit: calendar URL doesn't load that calendar

Reply:
interesting, it brings me to my calendar and says i've loaded it already.
i've updated the link to a calendar view.

Reply:
nit part deux 😀: The calendar title says "Public", but the link appears to be only available to company accounts. (I'm asked to auth when attempting to view in a private window.)